documentation/Using-the-SDK/Advanced/Adding-Column-Variants.md

```diff
@@ -0,0 +1,341 @@
+# Abstract
+
+This document outlines how to configure a [column](../../Glossary.md#column) with [column variants](../../Glossary.md#column-variant). Additionally, this document provides guidelines on when to use each type of column variant and when you should consider making create separate columns instead of column variants.
+
+# Motivation
+
+There are often instances where a single piece of data can be expressed in multiple ways. Prior to SDK version `1.3`, table authors were forced to add one column for each way the data can be displayed.
+
+For example, suppose a table has a "Process" column whose data identifies which process is associated with each row. This data can be expressed in multiple ways:
+* The process name
+* The process identifier (PID)
+* The process name + PID
+
+Without column variants, the table author would have to add three independent columns to a table to expose these three representations of the same data. With column variants, however, the table author can add one "Process" column with three variants - one for each representation.
+
+# When to Use Column Variants
```

Comment:
This explains the restrictions but doesn't really say when should one use column variants

Reply:
Added a sentence